Objects/typeobject.c

```diff
@@ -3493,11 +3488,11 @@ get_bases_tuple(PyObject *bases_in, PyType_Spec *spec)
 }
 
 static inline int
-check_basicsize_includes_size_and_offsets(PyTypeObject* type)
+type_check_basicsize_includes_size_and_offsets(PyTypeObject* type)
```

Comment:
I suggest to use a shorter name (it's just more convenient), but you can add a comment listing the few checks done by the function.

```suggestion
type_check_basicsize(PyTypeObject* type)
```